src/Transport/Topology/TypesScanner/AssemblyTypesScanner.cs

```diff
@@ -8,12 +8,7 @@ class AssemblyTypesScanner : ITypesScanner, IEquatable<AssemblyTypesScanner>
     {
         public AssemblyTypesScanner(Assembly assembly)
         {
-            if (assembly == null)
-            {
-                throw new ArgumentNullException(nameof(assembly), "It's not possible to initialize an AssemblyTypesScanner without specifing an assembly");
-            }
-
-            this.assembly = assembly;
+            this.assembly = assembly ?? throw new ArgumentNullException(nameof(assembly), "It's not possible to initialize an AssemblyTypesScanner without specifing an assembly");
```

Comment:
Was there a rule that required this change?

Reply:
Yes, IDE0016: Null check can be simplified.
And I agree with the change. It's good.